src/args.rs

```diff
@@ -1,8 +1,14 @@
-#[derive(Debug)]
-pub struct Args {}
+use clap::Parser;
 
-impl Args {
-    pub fn parse() -> Self {
-        todo!()
-    }
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+pub struct Args {
+	/// Application name
+	#[arg(long)]
+	app: String,
+	
+	/// Most recent time interval in milliseconds
+	#[arg(short, long, default_value_t = 1000)]
+	interval: u32,
```

Comment:
I think `-i` might be a little ambiguous here---to me, `-i` says 'input.' I don't think this option will need to be specified too frequently, so it's probably clearer to make it a `long` arg

Reply:
Sounds good! :) Changed